File: Fizz-Buzz/FizzBuzz.java
import java.util.Scanner;

public class FizzBuzz {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Enter no. to view FizzBuzz : ");
        int num = input.nextInt();

        for (int i = 1; i <= num ; i++)
        {
            if (i%5 == 0 && i%3 == 0)
                System.out.println("FizzBuzz");
            else if (i%5==0)
                System.out.println("Buzz");
            else if (i%3==0)
                System.out.println("Fizz");
            else
                System.out.println(i);
        }
    }
}

File: Fizz-Buzz/FizzBuzz.kt
import java.util.*

object FizzBuzz {
    @JvmStatic
    fun main(args: Array<String>) {
        val input = Scanner(System.`in`)
        println("Enter no. to view FizzBuzz : ")
        val nums = input.nextInt()
        for (i in 1..nums) {
            if (i % 5 == 0 && i % 3 == 0) println("FizzBuzz") else if (i % 5 == 0) println("Buzz") else if (i % 3 == 0) println("Fizz") else println(i)
        }
    }
}